_versions/2021-10-05-opensearch-1.1.0.markdown

```diff
@@ -92,4 +92,4 @@ pretty:
     opensearch-dashboards-min: 'OpenSearch Dashboards Minimum'
 
 ---
-OpenSearch is open source software that uses the Apache License version 2 (ALv2). ALv2 grants you well-understood usage rights; you can use, modify, extend, embed, monetize, resell, and offer OpenSearch as part of your products and services. The source for the entire project is available on [GitHub](https://github.com/opensearch-project/) and you're welcome to build from source for customized deployments. Downloadable artifacts for OpenSearch and OpenSearch Dashboards include plugins and tools, ready for you to use with minimal configuration.
\ No newline at end of file
+OpenSearch is open source software that uses the Apache License version 2 (ALv2). ALv2 grants you well-understood usage rights; you can use, modify, extend, embed, monetize, resell, and offer OpenSearch as part of your products and services. The source for the entire project is available on [GitHub](https://github.com/opensearch-project/) and you're welcome to build from source for customized deployments. Downloadable artifacts for OpenSearch and OpenSearch Dashboards include plugins and tools, ready for you to use with minimal configuration.
```

Comment:
Did you mean to edit this section? I see a few changes related to it.

Reply:
No. Looks like that's a mistake. Let me check.

Reply:
Looks like it's just a "newline" at the end of the line. Looks like my editor added a newline at the end. Is that a big deal?